boilerplates/react-union-boilerplate-monorepo/packages/app-sample/src/routes.js

```diff
@@ -1,17 +1,12 @@
-import loadHeroWidget from '@union-app/widget-hero';
-import loadContentWidget from '@union-app/widget-content';
+import universal from 'react-universal-component';
 
 export default [
 	{
 		path: 'hero',
-		getComponent: done => {
-			loadHeroWidget(mod => done(mod.default));
-		},
+		component: universal(import('@union-app/widget-hero')),
```

Comment:
You're sort of right. I decided to go with react-universal-component, which needs a [babel plugin](https://github.com/faceyspacey/babel-plugin-universal-import) to work properly... I forgot to add the plugin to our preset (sorry!). React-union is still usable without our scripts and babel preset (either by manual configuration, or using some worse solution, such as React Loadable).

The babel plugin will transform this so it will work as intended, although it might seem weird. I'm not sure if I should change it to the longer syntax, because the library author recommend this.